src/sentry/relay/config.py

```diff
@@ -345,3 +349,70 @@ def _filter_option_to_config_setting(flt, setting):
                 # ret_val['options'] = setting.split(' ')
                 ret_val["options"] = list(setting)
     return ret_val
+
+
+# Note: names for extracted metrics start with `measurement.`, while the
+# in-event keys start with `measurements.`
+ALL_MEASUREMENT_METRICS = frozenset(
+    [
+        "measurement.fp",
+        "measurement.fcp",
+        "measurement.lcp",
+        "measurement.fid",
+        "measurement.cls",
+        "measurement.ttfb",
+        "measurement.ttfb.requesttime",
+        "measurement.app_start_cold",
+        "measurement.app_start_warm",
+        "measurement.frames_total",
+        "measurement.frames_slow",
+        "measurement.frames_frozen",
+        "measurement.frames_slow_rate",
+        "measurement.frames_frozen_rate",
+        "measurement.stall_count",
+        "measurement.stall_total_time",
+        "measurement.stall_longest_time",
+        "measurement.stall_percentage",
```

Comment:
@untitaker Can we add `exclusive_time` that can be extracted from `contexts.trace.exclusive_time`?

Reply:
Nevermind. Ignore